app/assets/javascripts/discourse/components/post-menu.js.es6

```diff
@@ -349,6 +349,25 @@ const PostMenuComponent = Ember.Component.extend(StringBuffer, {
     this.sendAction('toggleBookmark', post);
   },
 
+  // Wiki button
+  buttonForWiki(post) {
+    if (!post.get('can_wiki')) return;
+
+    let buttonClass = 'wiki',
+        tooltip = 'post.controls.wiki';
```

Comment:
`buttonClass` can't be moved because I am concating the class. Eliminated the `tooltip` variable by returning the button from `if/else` block.